### DISCUSSION TITLE
Recommend quick edit to setup step (Small nitpick)

### DISCUSSION THREAD

**Discussion by ainglimo-personal** [mbround18/valheim-docker#745] [Instructions]
## Recommending quick edit to setup step
https://github.com/mbround18/valheim-docker/discussions/28#discussion-3188211

Just a couple of points which may cause less experienced engineers to get confused.

## Add user to docker group
I had to include the group I was adding my user to. This command could be simplified like this
"Make sure they have access to docker `usermod -aG [your users name]`"
to
"Make sure they have access to docker `usermod -aG docker ${USER}`"

This adds the current user that we are currently logged into to the `docker` group. Currently, the command fails because `-G` expects an argument

(I'm currently building this out on an Amazon Linux 2023 EC2 instance and seeing if I can get it to work)

**Comment by mbround18** [comment]
Thank you added! :)
